robottelo/bz_helpers.py

```diff
@@ -10,7 +10,7 @@
 from robozilla.parser import Parser
 
 BASE_PATH = os.path.join(get_project_root(), 'tests', 'foreman')
-VFLAGS = ['sat-{0}.{1}.{2}'.format(6, m, p) for m in '01234' for p in '0z']
+VFLAGS = ['sat-{0}.{1}.{2}'.format(6, m, p) for m in '012345' for p in '0z']
```

Comment:
So it's gonna strike us again in 6.6? can't we just update it to `0...9`?

Reply:
yeah that's what I was thinking too, of course we also get bitten when we go to 7.x but that's a long way off I guess.

Reply:
@abalakh , @flatfender,  for the moment we needed a fast fix, and yes of course we should generalize the solution.